hack/images/kubeone-e2e/install-kube-tests-binaries.sh

```diff
@@ -20,7 +20,7 @@ declare -A full_versions
 full_versions["1.16"]="v1.16.14"
 full_versions["1.17"]="v1.17.11"
 full_versions["1.18"]="v1.18.8"
-full_versions["1.19"]="v1.19.0"
+full_versions["1.19"]="v1.19.2"
```

Comment:
Can you also update other versions?

Reply:
done

Reply:
Can you also update versions in `.prow.yaml`?

Reply:
👍